test/runtests.jl

```diff
@@ -80,63 +101,72 @@ end
   Base.cumulative_compile_timing(true)
 end
 
-# Used in both Rings/slpolys.jl and StraightLinePrograms/runtests.jl
-const SLP = Oscar.StraightLinePrograms
+println("Making test list")
 
-include("Aqua.jl")
+testlist = []
+push!(testlist, "Aqua.jl")
 
-include("printing.jl")
+push!(testlist, "printing.jl")
 
-include("PolyhedralGeometry/runtests.jl")
-include("Combinatorics/runtests.jl")
+push!(testlist, "PolyhedralGeometry/runtests.jl")
+push!(testlist, "Combinatorics/runtests.jl")
 
-include("GAP/runtests.jl")
-include("Groups/runtests.jl")
+push!(testlist, "GAP/runtests.jl")
+push!(testlist, "Groups/runtests.jl")
 
-include("Rings/runtests.jl")
+push!(testlist, "Rings/runtests.jl")
 
-include("NumberTheory/nmbthy.jl")
+push!(testlist, "NumberTheory/nmbthy.jl")
 
 if Oscar.is_dev
-  include("Experimental/GITFans.jl")
+  push!(testlist, "Experimental/GITFans.jl")
```

Comment:
This conditional `push!` would be an obstacle to declaring `const testlist = [ ... ]`, but luckily it will be removed in PR #2416.